plasma/plasma-mvp/morevp-alice-bob.md

```diff
@@ -0,0 +1,103 @@
+MoreVP Alice-Bob Scenarios
+---
+
+Note that these scenarios work with N participants, not just limited to 1 input 1 output. 
+
+Also, note that in all of these scenarios the exits are given the priority of the youngest input to the transaction.
+
+#### Alice & Bob are honest and cooperating:
+
+Happy path.
+
+1. Alice spends UTXO1 in TX1 to Bob.
+2. Operator withholds, neither Alice nor Bob know if the transaction has been included. 
+3. Someone with access to TX1 (Alice, Bob, or otherwise) starts an exit and places `exit bond`.
+4. Alice and Bob "piggyback" onto the exit and each put up `piggyback bond`.
+5. After 1 week, Bob receives the money as if the transaction were included. All bonds are refunded.
+
+
+#### Bob spends:
+
+Bob trying to cheat.
+
+1. Alice spends UTXO1 in TX1 to Bob, creating UTXO2.
+2. TX1 is included in block N.
+3. Bob spends UTXO2 in TX2.
+4. TX2 is included in block N+M.
+5. Bob starts an exit from TX1.
+6. Bob "piggybacks" onto the exit and places `piggyback bond`
+7. Someone reveals TX2 spending UTXO2, so Bob's `piggyback bond` is slashed.
+
+
+#### Alice double spends:
+
+This is the case where Alice is malicious and has a double-spend secretly included before her transaction to Bob.
+
+1. Alice spends UTXO1 in TX1 to Bob.
+2. Operator withholds, neither Alice nor Bob know if the transaction has been included. 
+3. Unbeknownst to Bob, Alice has double spent UTXO1 in TX2. TX2 is included in the withheld block, but TX1 isn't.
+3. Someone with access to TX1 (Alice, Bob, or otherwise) starts an exit and puts up `exit bond`.
+4. Alice and Bob "piggyback" onto the exit and each put up `piggyback bond`.
+5. Within 3.5 days, the operator reveals TX2. 
+6. No one is able to reveal TX1 in response, so Bob's `piggyback bond` is refunded and Bob doesn't get to exit. Alice's `piggyback bond` is slashed. Operator gets `exit bond` for (5). 
```

Comment:
> No one is able to reveal TX1 in response,

why? Alice and operator can.

What if they did reveal TX1 then?

Reply:
`TX1` needs to be included in the chain before `TX2` to be revealed at this step. This is not the case, so no one can respond. The idea here is that if `TX1` is shown to be in the chain before `TX2`, then `TX1` must be the canonical spend.

Reply:
ok, so it's just the wording that is perhaps a little confusing, since TX1 has been revealed (for instance, during exit start). It's more a matter of TX1 not being able to be exited from, since TX2 invalidates it.

Reply:
Correct. TX2 invalidates TX1 because TX2 is included in the chain before TX1.